### ISSUE TITLE
wps_detect example not working on Windows

### ISSUE THREAD

**Issue by Ohmnivore** [mfontanini/libtins#132]
![](http://i.snag.gy/QCqOz.jpg)

I'm using a .dll build on Windows 7, VS 2013. I have winpcap 4.1.3 running. I tried using both my built-in Broadcom 4313GN and my USB TL-WN722N.

**Comment by mfontanini** [comment]
Can you try using an exanple that doesn't involve using wireless cards?
Like the traceroute one

**Comment by Ohmnivore** [comment]
The traceroute example shows the same popup. So my .dll or the linking process somewhere are nuts.

**Comment by mfontanini** [comment]
I just tried it and it works for me. Apparently operator<< for IPv4Address is not being exported on the DLL so I had to change that to build the traceroute example, but then it worked. I also tried dns_queries and it works fine too.

I just ran cmake disabling WPA2 decryption (mainly because I don't have openssl installed) and then built using msbuild. Finally, copied tins.dll into windows/system32 and everything worked as expected. I'm using VC 2013 on windows 7, just in case.
